listener/bsc_listener.go

```diff
@@ -68,7 +68,13 @@ func (l *BSCListener) poll() error {
 		if nextHeight <= latestPolledBlockHeight {
 			nextHeight = latestPolledBlockHeight + 1
 		}
-		latestBlockHeight, err := l.bscExecutor.GetLatestFinalizedBlockHeightWithRetry()
+		var latestBlockHeight uint64
+		if l.isOpCrossChain() {
+			// currently Get finalized block is not support by OPBNB yet
+			latestBlockHeight, err = l.bscExecutor.GetLatestBlockHeightWithRetry()
```

Comment:
do  we need to add more blocks for finality of opbnb

Reply:
This is done is the voteProcessor, num of blocks will be waited before broadcasting vote

Reply:
for opbnb, we consider it as one block finality.